ecosystem/cli/ci.py

```diff
@@ -17,37 +19,20 @@ class CliCI:
     Ex: `python manager.py ci parser_issue --body="<SOME_MARKDOWN>"`
     """
 
-    def __init__(self, root_path: Optional[str] = None):
-        """CliCI class."""
-        self.current_dir = root_path or os.path.abspath(os.getcwd())
-        self.resources_dir = "{}/ecosystem/resources".format(self.current_dir)
-        self.dao = DAO(path=self.resources_dir)
-        self.logger = logger
-
     @staticmethod
-    def parser_issue(body: str) -> None:
-        """Command for calling body issue parsing function.
+    def add_member_from_issue(body: str, resources_dir: str | None = None) -> None:
+        """Parse an issue created from the issue template and add the member to the database
 
         Args:
             body: body of the created issue
+            resources_dir: (For testing) Path to the working directory
 
         Returns:
-            logs output
-            We want to give the result of the parsing issue to the GitHub action
+            None (side effect is updating database)
         """
 
-        parsed_result = parse_submission_issue(body)
+        resources_dir = Path(resources_dir or (Path.cwd() / "ecosystem/resources"))
```

Comment:
Readability nit:

```suggestion
        resources_dir = Path(resources_dir) or (Path.cwd() / "ecosystem/resources")
```

Reply:
I think this will error if `resources_dir` is `None`